### PR TITLE
docs: Clarify committer vote procedures

### PR DESCRIPTION
**Pull request by joestringer**
A couple of questions have come up recently around how to handle
communication for voting processes for committer access. Improve the
docs to make these steps more clear for subsquent nominations and votes.

### CODE REVIEW

**Review by lizrice — APPROVED** [review]

**Review by sayboras — APPROVED** [review]

File: Documentation/community/governance/commit_access.rst [hunk 1]
```diff
@@ -383,6 +385,9 @@ Vote to Grant Commit Access
 Vote Results for Grant of Commit Access
 ---------------------------------------
 
+Vote results should be reported based on the vote count, ie after applying the
```
Comment (qmonnet) [inline Documentation/community/governance/commit_access.rst]:
```suggestion
Vote results should be reported based on the vote count, i.e. after applying the
```

File: Documentation/community/governance/commit_access.rst [hunk 2]
```diff
@@ -490,6 +495,9 @@ Vote to Revoke Commit Access
 Vote Results for Revocation of Commit Access
 --------------------------------------------
 
+Vote results should be reported based on the vote count, ie after applying the
```
Comment (qmonnet) [inline Documentation/community/governance/commit_access.rst]:
```suggestion
Vote results should be reported based on the vote count, i.e. after applying the
```

File: Documentation/community/governance/commit_access.rst [hunk 3]
```diff
@@ -186,11 +186,11 @@ The process to grant commit access to a candidate is simple:
 
 -  If the candidate agrees access is granted by setting up commit access
    to the repos. The new committer is invited to the #committers Slack channel,
-   *after* the nomination poll and related discussions have been deleted. The
-   name of the new committer is also added to the list in the MAINTAINERS.md_
-   file. The new committer is also added to the CNCF's list of Cilium
-   maintainers as documented `here
-   <https://github.com/cncf/foundation/blob/main/README.md#other-content>`__. 
+   *after* the nomination poll and related discussions have been deleted. There
```
Comment (qmonnet) [inline Documentation/community/governance/commit_access.rst]: Should we also explicitly state the motivation for deleting the poll result and related discussions?

**Review by qmonnet — APPROVED** [review]
Thanks for this!

**Review by qmonnet — CHANGES_REQUESTED** [review]
Spellchecker complains on `reposted`. Should be ready to merge after this is addressed.

**Pull request merged**